ChangeLog

```diff
@@ -1,5 +1,38 @@
 ### 3.9.1dev <- NOTE: The release version number will be 3.10.0 ###
 
+- Build: Updated bundled Qt6 to version 6.4.0 (#2892).
+  (contributed by @github-actions)
+
+- Build: Mac: Keep the signing-related keychain unlocked (#2927).
+  (contributed by @hoffie)
+
+- Build: Updated bundled JACK (Windows-only) to version 1.9.21.20221111 (#2960).
+  (contributed by @github-actions)
+
+- Build: Updated bundled Qt6 to version 6.4.1 (#2961).
+  (contributed by @github-actions)
+
+- Build: Updated bundled Qt6 to version 6.4.2 (#2991).
+  (contributed by @github-actions)
+
+- Build: Updated bundled Qt6 to version 6.4.3 (#3025).
+  (contributed by @github-actions)
+
+- Build: Updated bundled Qt6 to version 6.5.1 (#3070).
+  (contributed by @github-actions)
+
+- Docs: Added JSON-RPC server connection tips (#3101).
```

Comment:
Could you please check if this isn't already mentioned?

Reply:
This was also added by the `tools/changelog-helper.sh` script. I believe it checks whether the PR numbers are already in the ChangeLog. I'll check manually.

Reply:
There's no existing mention in the ChangeLog